### ISSUE TITLE
System tray icon battery % indication text renders improperly

### ISSUE THREAD

**Issue by EwanGreen4** [steam3d/MagicPods-Windows#282]
The system tray icon does not properly render the battery indication text. I assume that intended behavior would make it vertically centered and have anti-aliased text...
![image](https://user-images.githubusercontent.com/36348486/210039545-15108d59-2aa0-4362-8136-3ba0a6e568ae.png)

**Comment by steam3d** [comment]
Hi. It is right position. I'm using integers so that the icon doesn't blur. Because of this it is not possible to perfectly center. If the size of the number is odd, then an additional pixel will appear on one of the sides.

![image](https://user-images.githubusercontent.com/30986874/210042586-c7650387-05e0-4c1b-b462-e9aae2ff98de.png)

I can add anti-aliased filter, but users prefer more sharp and large digit

**Comment by EwanGreen4** [comment]
Considering other text in the taskbar *is* antialiased (see: the locale icon), I think default the behavior should to antialias. I'm sure users wouldn't be opposed to a toggle in the settings.
My issue is only seen in the preview build at the time of writing, where the XAML taskbar tray icons behave differently (the biggest difference is that they are larger).
The gap also looks like more than a pixel to me -- maybe it is an epx you are adding on either end. Have you considered using the same font size as the rest of the taskbar does, or using text metrics to find the best font size that your drawable region will allow?

**Comment by steam3d** [comment]
I use the Windows 11 Pro (not insider). I spent enough time to choose the best option. It also support different DPI up to 300%. I use **Microsoft Sans Serif** because it fits as much as possible free space.

I did not find the font which Windows task bar uses. It looks like Segoe UI but when I tried it - it looks differently. 

I will wait when Windows release new version to public because if it changes tray icon, I need to create two options, one for Windows 10 and second for Windows 11.

The trouble may be also in hard-code size of tray icon. 
![image](https://user-images.githubusercontent.com/30986874/210047747-56a7f846-6f2b-4d80-ac53-83f51ad1abc2.png)

**Comment by EwanGreen4** [comment]
Newer builds of Windows 11 use Segoe UI Variable. This may be of some use to you:
https://learn.microsoft.com/en-us/windows/apps/design/signature-experiences/typography#type-ramp

I wouldn't personally treat it as an icon... just drawable space you know &/ can determine the dimensions of, just like anything else. You are probably right in all your decisions thus far concerning font sizes and positions, but I have to wonder if you have yet brought in any library like Win2D that can draw to any old buffer using WinUI's contexts for things like fonts. Something to consider

**Comment by steam3d** [comment]
I am not sure that I understood. I do not use WinUI, because WPF does not support it. WPF has enough features to render text so this is a style question not technical. 

It is render on 4k screen Segoe UI Variable + anti-aliased. I forgot apply centered I think it does not matter for now. (Left - normal weight, right - semibold) 
![image](https://user-images.githubusercontent.com/30986874/210052601-2bde9db8-a705-44b3-8d79-910f5158101d.png)

**Comment by EwanGreen4** [comment]
It was a bad assumption on my part that you were using WinUI. 
I think the far right example looks much better. I also think that if you centered it, it would look even better

**Comment by steam3d** [comment]
@EwanGreen4 WinUI 3 is very raw technology.  I added a task to improve tray icon. I could not say when it will be available, because I have plenty tasks right now.

**Comment by steam3d** [comment]
I made some tests.

Left - Segoe UI Variable Text Semibold
Right - Microsoft Sans Serif

![image](https://user-images.githubusercontent.com/30986874/212913308-25a2d761-989e-4d65-aa76-23dee351cbdb.png)

The right one has better readability.

I made absolute the same size and font style
![image](https://user-images.githubusercontent.com/30986874/212914119-f899f3b2-8b56-4625-af86-1e60755be343.png)

I will fix the center issue and enable anti-aliased. I try to fill square with the text as mach as possible to get more  readability of text.
